src/batched/KokkosBatched_Gemm_TeamVector_Impl.hpp

```diff
@@ -138,7 +138,7 @@ namespace KokkosBatched {
            const CViewType &C) {
       // C = beta C + alpha A B
       // C (m x n), A(m x k), B(k x n)
-      return TeamGemmInternal<Algo::Gemm::Unblocked>::
+      return TeamVectorGemmInternal<Algo::Gemm::Unblocked>::
```

Comment:
@kyungjoo-kim: Was this just a typo or is there a reason to be calling `TeamGemm` instead of `TeamVectorGemm` here?

Reply:
typo.